vybetoken/dashboard
# Claim button enabled after canceled transaction

Github Issue: 

https://github.com/vybetoken/dashboard/issues/2

How to test:

Connect with MetaMask
Click "Claim" Button
Reject Transaction

You should be able to repeat this process because the Claim button should be enabled after rejecting the transaction
